release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,348 @@
+This enhancement helps you identify the specific IP addresses your deployments are communicating with and gives you better insight into the external interactions and potential risks.
+
+//ROX-25007
+[id="enhanced-options-for-the-roxctl-netpol-generate-command_{context}"]
+=== Enhanced options for the roxctl netpol generate command
+
+{product-title-short} 4.7 introduces improved options for the `roxctl netpol generate` command. You can now specify DNS ports by using names as well as numbers. If you do not specify a port, port `53` is used automatically. 
+
+However, you can change this value by using the `--dnsport` flag. For example, you can now specify the port when a service has a defined name by using the `--dnsport dns`. 
+If you are using {osp} with the default DNS setting, you can change the default port by using the `--dnsport` flag, as {osp}’s DNS pod listens on port `5353`.
+
+For more information, see xref:../cli/command-reference/roxctl-netpol.adoc#roxctl-netpol-generate_roxctl-netpol[roxctl netpol generate].
+
+//ROX-24589
+//tech preview
+[id="integrate-the-vulnerability-insights-into-the-red-hat-developer-hub_{context}"]
+=== Integrate the vulnerability insights into the Red{nbsp}Hat Developer Hub
+
+:FeatureName: Integration of vulnerability insights into the Red{nbsp}Hat Developer Hub
+include::snippets/technology-preview.adoc[]
+
+In {product-title-short} 4.7, you can now bring vulnerability insights directly into the Red{nbsp}Hat Developer Hub and other platforms based on the upstream Backstage project by using the Backstage Plugin.
+You can incorporate the {product-title-short} plugin into your developer environment and review key vulnerabilities for your applications within the platform and take the necessary remediation actions.
+
+//ROX-24052
+//tech preview
+[id="generate-sboms-from-the-scanned-container-images_{context}"]
+=== Generate SBOMs from the scanned container images
+
+:FeatureName: Generation of SBOMs from the scanned container images
+include::snippets/technology-preview.adoc[]
+
+With {product-title-short} 4.7, you can now generate Software Bill of Materials (SBOMs) from the scanned container images. These SBOMs give you a detailed overview of all software components, dependencies and libraries within your application.
+{product-title-short} creates SBOMs of the Analyzed type and complies with the Software Package Data Exchange (SPDX) 2.3 specification.
+
+For more information about the SBOMs of the Analyzed type, see
+link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[Types of SBOM Documents] (America's Cyber Defense Agency documentation).
+
+//ROX-17535
+[id="automatic-certificate-rotation-by-using-the-mtls-architecture_{context}"]
+=== Automatic certificate rotation by using the mTLS architecture
+
+{product-title-short} 4.7 introduces certificate-based architecture (mTLS) for communication between your secured cluster components and Central. 
```

Comment:
I think this is misleading, mTLS was always used for communication between Secured Clusters and Central.

What is new is that the Secured Cluster certificates are automatically rotated. Previously, new credentials had to be generated on Central side, and manually applied to each Secured Cluster.

Reply:
I was just about to point this out too :)  mTLS has always been there, now we have ability to auto-rotate the certs